pkg/hostagent/requirements.go

```diff
@@ -132,6 +132,20 @@ Also see "/var/log/cloud-init-output.log" in the guest.
 A possible workaround is to run "lima-guestagent install-systemd" in the guest.
 `,
 	})
+	if *a.y.SSH.ForwardAgent {
+		req = append(req, requirement{
+			description: "linking ssh auth socket to /tmp/ssh-auth/agent.sock",
+			script: `#!/bin/bash
+set -eux -o pipefail
+if ! timeout 30s bash -c "mkdir -p -m 700 /tmp/ssh-auth && ln -f $SSH_AUTH_SOCK /tmp/ssh-auth/agent.sock"; then
```

Comment:
I haven't tested it, but I think you need to remove the old socket before linking it again, in case `/tmp` has not been cleared by the reboot:

```suggestion
if ! timeout 30s bash -c "mkdir -p -m 700 /tmp/ssh-auth && rm -f /tmp/ssh-auth/agent.sock && ln -f $SSH_AUTH_SOCK /tmp/ssh-auth/agent.sock"; then
```

Otherwise I would expect you to get a `ln: /tmp/ssh-auth/agent.sock: File exists` error.

Reply:
The `-f` argument removes any existing files.

```
-f, --force
              remove existing destination files
```

Reply:
I've tried stopping the VM and and starting it again and it works.